lib/experimental/fonts/font-library/font-library.php

```diff
@@ -57,6 +57,27 @@ function wp_register_font_collection( $config ) {
 	}
 }
 
+if ( ! function_exists( 'sanitize_font_family_content' ) ) {
+	/**
+	 * Sanitize font family content.
+	 *
+	 * @param array $data                An array of slashed and processed post data.
+	 * @param array $postarr             An array of sanitized (and slashed) but otherwise unmodified post data.
+	 * @param array $unsanitized_postarr An array of slashed yet *unsanitized* and unprocessed post data as originally passed to wp_insert_post().
+	 *
+	 * @return array The post data that will be inserted in the database.
+	 */
+	function sanitize_font_family_content( $data, $postarr, $unsanitized_postarr ) {
```

Comment:
We're assuming that `$unsanitized_postarr` exists and is valid. Maybe we should add a check here as well.

Reply:
Done in https://github.com/WordPress/gutenberg/pull/56172/commits/832db93b962481affb41d21cd8b41ab479634f59.